src/modules/bets.js

```diff
@@ -0,0 +1,331 @@
+        const percentage = optionVotes / votesCount * 100 || 0;
+
+        return i18n('bet.line', {
+            option,
+            optionVotes,
+            percentage,
+        });
+    }).join('');
+    response.output += output;
+    return response;
+};
+
+const listBets = async function (response, {
+    i18n,
+    getModuleData,
+}) {
+    const { betsList = [], votesListBets = [] } = await getModuleData('bets');
+
+    if (!betsList.find(bet => bet.isOpen)) {
+        response.output = i18n('bet.none');
+        return response;
+    }
+
+    response.output = `${i18n('bets.list')}\n`;
+
+    betsList.filter(bet => bet.isOpen).forEach((bet) => {
+        const votesCount = votesListBets.filter(vote => vote.betId === bet.betId).length;
+        const {
+            dateCreated,
+            questionBet,
+            options,
+            betId,
+        } = bet;
+
+        const date = moment(dateCreated).format('DD/MM');
+
+        let output = i18n('bet.header', {
+            date,
+            questionBet,
+            votesCount,
+            betId,
+        });
+
+        output += options.map((option) => {
+            const optionVotes = votesListBets.filter(vote => (vote.betId === betId && vote.option === bet.options.indexOf(option))).length;
+            const percentage = optionVotes / votesCount * 100 || 0;
+
+            return i18n('bet.line', {
+                option,
+                optionVotes,
+                percentage,
+            });
+        }).join('');
+
+        response.output += `${output}\n`;
+    });
+    return response;
+};
+
+const closeBet = async function (response, {
+    i18n,
+    getModuleData,
+    updateModuleData,
+}) {
+    const { args: { requestedBetId, winOption } } = response;
+    const { betsList = [], votesListBets = [] } = await getModuleData('bets');
+    const currentBet = betsList.find(bet => bet.betId === requestedBetId);
+    const winList = votesListBets.filter(
+        vote => vote.betId === requestedBetId && vote.option === winOption - 1,
+    );
+
+    if (!betsList.find(bet => bet.isOpen)) {
+        response.output = i18n('bet.none');
+        return response;
+    }
+
+    if (!currentBet) {
+        response.output = i18n('bet.notFound', { requestedBetId });
+        return response;
+    }
+
+    if (!currentBet.isOpen) {
+        response.output = i18n('bet.alreadyClosed');
+        return response;
+    }
+
+    const newList = betsList.filter(bet => bet.betId !== requestedBetId);
+    currentBet.isOpen = false;
+    newList.push(currentBet);
+
+    if (!isEqual(betsList, newList)) {
+        updateModuleData('bets', {
+            betsList: newList,
+        });
+    }
+
+    if (winList.length === 0) {
+        response.output = i18n('bet.closeNoWinner', { requestedBetId });
+        return response;
+    }
+
+    const winners = winList.reduce((acc, winner) => {
+        let accum = acc;
```

Comment:
Why?
```js
winList.reduce((acc, winner) => acc + `<@${winner.voterId}>`, '');
```

Reply:
Because eslinter writes that it’s wrong. Google prompted to do this.

Reply:
The only thing eslint is complaining about the string before is string concatenation, so do 
```js
`${acc}, <@${winner.voterId}>`
```